x-pack/plugins/uptime/public/apps/uptime_overview_fetcher.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { fetchPingHistogram, fetchSnapshotCount } from '../state/api';
+import { UptimeFetchDataResponse } from '../../../observability/public/typings/fetch_data_response';
+
+export async function fetchUptimeOverviewData({ startTime, endTime, bucketSize }) {
```

Comment:
I noticed you're not using the `bucketSize`, could you tell me why? We want to show the same information in the `x-axis` for all charts in the Observability page.

Reply:
@cauemarcondes i have updated to use the bucketSize